pyevmasm/evmasm.py

```diff
@@ -1090,6 +596,288 @@ def assemble_hex(asmcode, pc=0, fork=DEFAULT_FORK):
         return '0x' + hexlify(b''.join([x.bytes for x in asmcode])).decode('ascii')
     return '0x' + hexlify(assemble(asmcode, pc=pc, fork=fork)).decode('ascii')
 
+
+class InstructionTable():
+    """
+    EVM Instruction factory
+        Implements an immutable, iterable instruction LUT that can be indexed by both mnemonic or opcode.
+
+    Example::
+
+        >>> from pyevmasm import instruction_tables
+        >>> instruction_table = instruction_tables['byzantium']
+        >>> instruction_table[0]
+        Instruction(0x0, 'STOP', 0, 0, 0, 0, 'Halts execution.', None, 0)
+        >>> instruction_table['STOP']
+        Instruction(0x0, 'STOP', 0, 0, 0, 0, 'Halts execution.', None, 0)
+        >>> i = instruction_table.__iter__()
```

Comment:
```
i = iter(instruction_table)
next(i)
next(i)
```

Reply:
I think the goal is to have it compatible with both pys.